server/session/chunk.go

```diff
@@ -126,13 +126,21 @@ func (s *Session) subChunkEntry(offset protocol.SubChunkOffset, ind int16, col *
 	return entry
 }
 
+// dimensionID  returns the dimension ID of the world that the session is in.
+func (s *Session) dimensionID() int32 {
+	d, _ := world.DimensionID(s.c.World().Dimension())
+	return int32(d)
+
```

Comment:
Would return this newline, otherwise looks good.

Reply:
fixed